model/model.py

```diff
@@ -133,6 +131,17 @@ def make_random_passenger(self, force: bool=False) -> None:
             p = Passenger(start_floor, end_floor)
             self._passengers.add_passenger(p)
 
+    def make_passengers_from_schedule(self) -> None:
+        _, event = self.level.get_next_event(self._move_count)
```

Comment:
minor: level ->  _level (private field).

Also get_next_level seems like it should just return an `event`, not `tuple(move, event)`